src/main/groovy/com/avioconsulting/mule/linter/rule/configuration/FlowSubflowNamingRule.groovy

```diff
@@ -0,0 +1,62 @@
+package com.avioconsulting.mule.linter.rule.configuration
+
+import com.avioconsulting.mule.linter.model.Application
+import com.avioconsulting.mule.linter.model.ConfigurationFile
+import com.avioconsulting.mule.linter.model.MuleComponent
+import com.avioconsulting.mule.linter.model.Rule
+import com.avioconsulting.mule.linter.model.RuleViolation
+
+class FlowSubflowNamingRule extends Rule {
+
+    enum CaseFormat {
```

Comment:
I would pull this logic out into a StringUtil or StringParser(?) I know Adam doesn't like naming classes 'Util'....

We would want to be able to check any string, against a format... so that it can be reused in many rules... 

Then we can write some tests that just parse strings, and you wouldn't need so many config files...

Reply:
done.